cluster-stamp.bicep

```
@@ -1269,6 +1270,96 @@ resource mc 'Microsoft.ContainerService/managedClusters@2022-10-02-preview' = {
   ]
 }
 
+resource dataCollectionRules_MSCI_cluster_syslog 'Microsoft.Insights/dataCollectionRules@2021-09-01-preview' = {
```

```suggestion
resource dcrClusterSyslog 'Microsoft.Insights/dataCollectionRules@2021-09-01-preview' = {
```

please let's try following the same/similar naming convention for new resources